packages/core/marine-worker/package.json

```diff
@@ -24,7 +24,7 @@
     "vitest": "0.34.6"
   },
   "dependencies": {
-    "@fluencelabs/marine-js": "0.7.2",
+    "@fluencelabs/marine-js": "0.8.0",
```

Comment:
Why do you need marine-js is so many places? I believe this is the 3rd place I see marine-js dependency.

Reply:
Yes, need to refactor it. Ideally we want to use it in marine-worker only